### PR TITLE
Al 1227 asiaq integration socify validate

### PR DESCRIPTION
**Pull request by rruster**

### CODE REVIEW

File: disco_aws_automation/socify_helper.py [hunk 1]
```diff
@@ -10,78 +10,117 @@
...
 
 
+SocifyConfig = {
+    "EVENT": {
+        "basePath": "/event",
+        "use_data": True
+    },
+    "VALIDATE": {
+        "basePath": "/validate",
+        "use_data": False
```
Comment (aoskotsky-amplify) [inline disco_aws_automation/socify_helper.py]: Why wouldn't we send data for the `/validate` call?

Comment (rruster) [inline disco_aws_automation/socify_helper.py]: For the moment, we do not have any.

**Comment by aoskotsky-amplify** [comment]
build

**Comment by rruster** [comment]
build

**Pull request merged**